content/en/docs/concepts/overview/working-with-objects/labels.md

```diff
@@ -78,7 +78,7 @@ Valid label value:
 
 * must be 63 characters or less (can be empty),
 * unless empty, must begin and end with an alphanumeric character (`[a-z0-9A-Z]`),
-* could contain dashes (`-`), underscores (`_`), dots (`.`), and alphanumerics between.
+* could contain dashes (`-`), underscores (`_`), dots (`.`), and alphanumerics allowed in between.
```

Comment:
IMO
I don't feel this updated text adding any value, current existing context looks fine.